docs/install.md

```diff
@@ -0,0 +1,203 @@
+# Installing Tekton Dashboard
+
+This guide explains how to install Tekton Dashboard.
+
+## Pre-requisites
+
+[Kubernetes](https://github.com/kubernetes/kubernetes) must be installed with version 1.15.0 or later if you want to use a version of Tekton Pipelines newer than v0.10.1.
+
+[Tekton Pipelines](https://github.com/tektoncd/pipeline) must be installed in order to use the Tekton Dashboard. Instructions to install Tekton Pipelines can be found [here](https://github.com/tektoncd/pipeline/blob/master/docs/install.md). 
+
+### Which version should I use?
+
+It is strongly recommended to use the *v0.6.1.4* release for Tekton Pipelines v0.11.x and v0.12.x. This is a critically important security release and can display components from Triggers 0.4.
```

Comment:
should this section be removed since you've already included the compatibility matrix in the main README?
or move it from the main README to here if this is content we're going to be publishing on the website? (potentially everything in the docs folder...)